extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugInterface.java

```diff
@@ -131,6 +132,7 @@ SmugMugImageUploadResponse uploadImage(
     headersMap.put("X-Smug-AlbumUri", albumUri);
     headersMap.put("X-Smug-ResponseType", "JSON");
     headersMap.put("X-Smug-Version", "v2");
+    headersMap.put("X-Smug-AlbumOverflow", "1");
```

Comment:
I didn't do this, where is it coming from? @seehamrun

Reply:
lets remove this